website/content/en/docs/building-operators/golang/testing.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Testing your Operator project
+linkTitle: Testing with EnvTest
+description: Learn how to ensure the quality of your Operator project 
+weight: 70
+---
+
+## Overview
+
+The Operator SDK project recommends controller-runtime's [envtest][envtest] because it has a more active contributor community, it has become more mature than Operator SDK's test framework, and it does not require an actual cluster to run tests, which can be a huge benefit in CI scenarios. 
+
+## Using EnvTest
+
+You will see that `controllers/suite_test.go` is created. This file contains boilerplate for executing integration tests using [envtest][envtest] with [ginkgo](https://onsi.github.io/ginkgo/) and [gomega](https://onsi.github.io/gomega/).
+
+It means that you will implement your controller's tests in go using this stack and it will be called by the `suite_test.go` via go tool such as: 
+
+```shell
+go test controllers/ -v -ginkgo.v
+```
+
+The projects generated by using the SDK tool has a Makefile which will contain the target `tests` that executes this tests by default when, for example, you run `make tests` your `make docker-build IMG=<some-registry>/<project-name>:<tag>`.  
+
+To learn more about how you can test your controllers, see the documentation about [writing controller tests][writing-controller-tests].
+
+**NOTE** SDK project adopted this stack to perform its tests. In this way you might be able to check the tests implemented for the tool as well which can help you such as an example of its possibilities. See that [controller-runtime](https://github.com/kubernetes-sigs/controller-runtime) is covered by tests using the same stack as well. 
+
+## Other Options
+
+Also, you can use a declaratively way that is provided by SDK to define black-box tests using the [kuttl](https://kuttl.dev/) project. In the project scaffolded by the tool, you can simply define YAML manifests that specify the expected before-and-after state of a cluster when one of the Operator’s services which is used. For more info check [Writing Kuttl Scorecard Tests][writing-kuttl-scorecard-tests].
```

Comment:
how about "Also you can write tests for your operator in a declarative way [kuttl](link). Using kuttl, you can define YAML manifests that specify the expected before and after statues of a cluster when your operator is used. For more info see [Writing Kuttl Scorecard Tests](link).

Reply:
Changed based on your feedback as well. Tks

Reply:
:+1: